sdk/cosmos/azure-cosmos/test/test_query.py

```diff
@@ -12,7 +12,7 @@
 import azure.cosmos.cosmos_client as cosmos_client
 import azure.cosmos.exceptions as exceptions
 import test_config
-from azure.cosmos import http_constants, DatabaseProxy
+from azure.cosmos import http_constants, DatabaseProxy,_retry_utility, _endpoint_discovery_retry_policy
```

Comment:
`_retry_utility` has already been imported in line 11. Please remove this and use `retry_utility` instead